src/uPlot.js

```diff
@@ -1658,29 +1658,38 @@ export default function uPlot(opts, data, then) {
 			}
 			else {
 				// setSelect should not be triggered on move events
-				let uni = drag.uni;
+				let dx = abs(mouseLeft0 - mouseLeft1);
+				let dy = abs(mouseTop0 - mouseTop1);
 
-				if (uni != null) {
-					let dx = abs(mouseLeft0 - mouseLeft1);
-					let dy = abs(mouseTop0 - mouseTop1);
+				let dragX = drag.x && dx >= drag.dist;
```

Comment:
so, we already have global dragX and dragY vars which these are masking - the name clash is confusing. would it not be possible to reuse the existing globals here? their purpose seems identical.